backend/coins/btc/account.go

```diff
@@ -475,12 +475,36 @@ func (account *Account) Notifier() accounts.Notifier {
 	return account.notifier
 }
 
+// FeeTargets represents an array of FeeTarget pointers.
+type FeeTargets []*FeeTarget
+
+// Highest returns the feeTarget with the highest fee.
+func (feeTargets FeeTargets) Highest() *FeeTarget {
+	if len(feeTargets) < 1 {
+		return nil
+	}
+
+	highestFeeIndex := 0
+	var highestFee int64
```

Comment:
i think the below loop does not work if all fee targets are nil.

you could fix it by having a `var highestFeeTarget *FeeTarget`, and in the loop do something simliar to this:

```
if feeTarget == nil || feeTarget.feeRatePerKb == nil {
			continue
}
if highestFeeTarget == nil || *feeTarget.feeRatePerKb > *highestFeeTarget.feeRatePerKb {
    highestFeeTarget = feeTarget
}
```

Reply:
nice catch, thanks!